projects/examples/src/components/datagrid/datagrid-detail-row.example.component.ts

```diff
@@ -40,3 +60,42 @@ export class DatagridDetailRowExampleComponent {
         };
     }
 }
+
+@Component({
+    selector: 'vcd-datagrid-detail-row-sub-example',
+    template: `
+        <div [clrLoading]="loading">
+            {{ JSON.stringify(config.record) }} : Record<br />
+            {{ config.index }} : Index<br />
+            {{ config.count }} : Count<br />
+        </div>
+    `,
+})
+export class DatagridDetailRowSubComponent implements OnInit, ComponentRenderer<DetailRowConfig<Data>> {
+    loading = true;
+    JSON = JSON;
+
+    @Input() config: DetailRowConfig<Data>;
+
+    ngOnInit(): void {
+        setTimeout(() => {
+            this.loading = false;
+        }, 2000);
+    }
+}
+
+@Component({
+    selector: 'vcd-datagrid-detail-row-sub-example',
+    template: `
+        <div>
+            {{ JSON.stringify(config.record) }} : Record<br />
```

Comment:
This seems backwards, I usually think key, value, key, value, this seems to be value, key

A simple HTML tag for key values is `<dl> <dt> <dd>` See https://developer.mozilla.org/en-US/docs/Web/HTML/Element/dl